cmd/gaia/app/genesis.go

```diff
@@ -75,23 +78,54 @@ func NewGenesisAccount(acc *auth.BaseAccount) GenesisAccount {
 	}
 }
 
-func NewGenesisAccountI(acc auth.Account) GenesisAccount {
+func NewGenesisVestingAccount(vacc auth.VestingAccount) GenesisAccount {
 	return GenesisAccount{
+		Address:       vacc.GetAddress(),
+		Coins:         vacc.GetCoins(),
+		AccountNumber: vacc.GetAccountNumber(),
+		Sequence:      vacc.GetSequence(),
+		Vesting:       true,
+		StartTime:     vacc.GetStartTime(),
+		EndTime:       vacc.GetEndTime(),
+	}
+}
+
+func NewGenesisAccountI(acc auth.Account) GenesisAccount {
```

Comment:
Why do we need this function and `NewGenesisVestingAccount` - can't we just call `NewGenesisAccountI` on an `auth.VestingAccount`?

Reply:
Ahhh indeed we can -- updated.